test/volumes-automount.test.js

```diff
@@ -0,0 +1,626 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * Copyright (c) 2017, Joyent, Inc.
+ */
+
+var assert = require('assert-plus');
+var child_process = require('child_process');
+var fs = require('fs');
+var test = require('tape').test;
+var vasync = require('vasync');
+var verror = require('verror');
+
+var common = require('./common');
+var libuuid = require('libuuid');
+var mod_config = require('../lib/config.js');
+var mod_testConfig = require('./lib/config');
+var mod_testNetworks = require('./lib/networks');
+var mod_testVolumes = require('./lib/volumes');
+
+var machinesCommon = require('./machines/common');
+
+var CONFIG = mod_config.configure();
+var IMGAPI_SOURCE = 'https://images.joyent.com';
+var KEY_FILENAME = '/tmp/cloudapi-test-key';
+var TEST_IMAGE_LX = '7b5981c4-1889-11e7-b4c5-3f3bdfc9b88b'; // ubuntu-16.04
```

Comment:
Is it safe to hardcode it?

Reply:
I think so for now, but I'll create a ticket to revisit this in the future.